moabb/evaluations/evaluations.py

```diff
@@ -85,10 +85,9 @@ def evaluate(self, dataset, subject, pipelines):
         if not event_id:
             raise(ValueError("Dataset had no selected events"))
 
-        sub = dataset.get_data([subject], stack_sessions=False)[0]
```

Comment:
This is why we got only one score per subject even when multiple sessions

Reply:
we don't -- I checked in the .csv and there are multiple scores for a single subject. This is because get_data always returns a list but we know that list only has one member here (a list corresponding to sessions for the given subject).

Reply:
yeah, this is actually another issue. I don't think we have mutiple score for the Openvibe MI dataset, but that's because it does not return the session correctly. I will try to fix it in another PR.